### PR TITLE
[DNM] pdctl: add show the stores with a specifying status

### PR DESCRIPTION
**Pull request by nolouch**
- support like "echo store | pd-ctl  "
- add store status command
- let store `capacity` `available` size human-readable
PTAL @siddontang @huachaohuang @overvenus

### CODE REVIEW

**Comment by siddontang** [comment]
>support like "echo store | pd-ctl "

why do we need to support this?

**Comment by huachaohuang** [comment]
Can you pretty print the output inside the server API directly? 
Otherwise we need to maintain both packages if we update the structure.

File: pdctl/command/global.go [hunk 1]
```diff
@@ -13,37 +13,88 @@
...
-	"fmt"
-	"io"
-	"net/http"
-	"net/url"
-	"os"
-
-	"github.com/spf13/cobra"
+import "fmt"
+
+const (
```
Comment (huachaohuang) [inline pdctl/command/global.go]: Can we use an external package to handle these things?

**Comment by nolouch** [comment]
@siddontang  support pipe, some people like using it in this way.
@huachaohuang  I had tried to do this, but the struct defined in proto.

File: cmd/pd-ctl/main.go [hunk 2]
```diff
@@ -49,6 +50,14 @@ func main() {
 		syscall.SIGTERM,
 		syscall.SIGQUIT)
 
+	var otherInput []string
```
Comment (siddontang) [inline cmd/pd-ctl/main.go]: s/otherInput/input/

File: pdctl/command/global.go (hunk 1 above)
Comment (siddontang) [inline pdctl/command/global.go]: I think we can use https://github.com/dustin/go-humanize directly.

Comment (nolouch) [inline pdctl/command/global.go]: got it

**Comment by nolouch** [comment]
I will send  a new PR for this

**Pull request closed without merging**